knowledge-base/window-backdrop-blur.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Apply a backdrop filter with a blur effect to the Modal Window background
```

Comment:
Use title case. See [Titles and Headings](https://docs.telerik.com/style-guide/titles-and-headings#capitalization).